prototype_source/inductor_cpp_wrapper_tutorial.rst

```diff
@@ -0,0 +1,88 @@
+Inductor cpp wrapper tutorial
+==============================================================
+
+**Author**: `Chunyuan Wu <https://github.com/chunyuan-w>`_, `Bin Bao <https://github.com/desertfire>`__, `Jiong Gong <https://github.com/jgong5>`__
+
+Prerequisites:
+------------
+-  `torch.compile and TorchInductor concepts in PyTorch <https://pytorch.org/tutorials/intermediate/torch_compile_tutorial.html>`__
+
+Introduction
+------------
+
```

Comment:
The intro doesn't really explain why this useful, you generate python code to invoke kernels in which case how does this reduce python overhead?

Would suggest adding a longer explaination here

Reply:
Added more details in the Introduction to state that the default Inductor generates python code while the proposed cpp wrapper produces cpp code in order to reduce the overhead of the interpreted language.